### PR TITLE
Slow log:  измерять медленные запросы в милисекундах

### PR DESCRIPTION
**Pull request by seeforschauer**

### CODE REVIEW

File: metrics/slow_query.go [hunk 1]
```diff
@@ -9,14 +9,14 @@ import (
 )
 
 type SlowQueryMetrics struct {
-	mutex   sync.Mutex
-	metrics map[string]prometheus.Gauge
-	seconds int
+	mutex        sync.Mutex
+	metrics      map[string]prometheus.Gauge
+	milliseconds int
```
Comment (AlekSi) [inline metrics/slow_query.go]: `time.Duration`?

File: postgresql_exporter.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ var (
 	listenAddress = flag.String("web.listen-address", ":9104", "Address to listen on for web interface and telemetry.")
 	metricPath    = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
 	databases     = flag.String("db.names", "", "Comma-separated list of monitored DB.")
-	slow          = flag.Int("db.consider-query-slow", 5, "Queries with execution time higher than this value will be considered as slow (in seconds).")
+	slow          = flag.Int("db.consider-query-slow", 100, "Queries with execution time higher than this value will be considered as slow (in miliseconds).")
```
Comment (AlekSi) [inline postgresql_exporter.go]: `flag.Duration`?

File: metrics/slow_query.go (hunk 1 above)
Comment (lename) [reply metrics/slow_query.go]: Сделала в main.

File: metrics/slow_query.go [hunk 3]
```diff
@@ -4,19 +4,20 @@ import (
...
 	metrics      map[string]prometheus.Gauge
-	milliseconds int
+	milliseconds int64
 }
 
-func NewSlowQueryMetrics(msToConsiderSlow int) *SlowQueryMetrics {
+func NewSlowQueryMetrics(durationToConsiderSlow time.Duration) *SlowQueryMetrics {
 	return &SlowQueryMetrics{
-		milliseconds: msToConsiderSlow,
+		milliseconds: durationToConsiderSlow.Nanoseconds() / 1000000,
```
Comment (AlekSi) [inline metrics/slow_query.go]: From docs:
> To count the number of units in a Duration, divide.

So `durationToConsiderSlow / time.Millisecond` is preferable.

File: metrics/slow_query.go [hunk 4]
```diff
@@ -4,19 +4,20 @@ import (
...
+	mutex        sync.Mutex
+	metrics      map[string]prometheus.Gauge
+	milliseconds int64
 }
 
-func NewSlowQueryMetrics(secondsToConsiderSlow int) *SlowQueryMetrics {
+func NewSlowQueryMetrics(durationToConsiderSlow time.Duration) *SlowQueryMetrics {
 	return &SlowQueryMetrics{
-		seconds: secondsToConsiderSlow,
+		milliseconds: durationToConsiderSlow.Nanoseconds() / int64(time.Millisecond),
```
Comment (AlekSi) [inline metrics/slow_query.go]: Sorry that I wasn't clear enough. Idiomatic form is
```go
int64(durationToConsiderSlow / time.Millisecond)
```
This way you get a number of units on the right from any duration on the left.
And I'm sorry for nitpicking. :)

Comment (AlekSi) [reply metrics/slow_query.go]: https://golang.org/pkg/time/#Duration

Comment (lename) [reply metrics/slow_query.go]: At least it's more beautiful.

**Pull request merged**